### ISSUE TITLE
Please consider reverting AppImages to be built on 18.04

### ISSUE THREAD

**Issue by Novum** [Novum/vkQuake#729]
Or update the glibc version notice in the release docs. Building AppImages on 20.04 or 22.04 will produce binaries that require significantly newer Linux versions and I honestly do not understand what the upside is.

**Comment by vsonnier** [comment]
Hi @Novum I've read your other note also. There is no rush to take a recent distrib as base indeed. Going back to 18.04 will need to revert part of e8cf9edb1117f9f7ac856cf35c9a54d4439a6c2b, but right now I reverted 20.04 => 22.04.

**Comment by vsonnier** [comment]
I should have made a branch for this sorry for the spam :)

**Comment by vsonnier** [comment]
Updated Releases description to specify 'at least 2.31'.

**Comment by bubbleguuum** [comment]
You can compile for an older version of the glibc with [glibc_version_header](https://github.com/wheybags/glibc_version_header). For example you can target v2.27 (Ubuntu 18.04) or older. I've used this to compile a static version of ffmpeg, except for the glibc requiring v2.2+.

**Comment by Novum** [comment]
It's honestly crazy that the toolchain is so tightly tied to the system version on Linux, it always irritated me.

**Comment by bubbleguuum** [comment]
Agreed. The should be an easy way to configure gcc or clang to target any older version.
